app/uploaders/file_uploader.rb

```diff
@@ -49,7 +49,10 @@ def cache_dir
         tmp = Tempfile.new 'op_uploaded_files'
         path = Pathname(tmp)
 
-        tmp.delete # delete temp file
+        tmp.unlink # delete temp file. In windows this fails silently.
```

Comment:
@scumie How about closing the file first before unlinking it? This way it should work on Windows too.